docs/core/linux-prerequisites.md

```diff
@@ -88,13 +103,14 @@ The installer bash script is used in automation scenarios and non-admin installa
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-## Install .NET Core dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
+## Install .NET Core 1.x dependencies for Red Hat Enterprise Linux (RHEL) 7 Server
```

Comment:
do you need 1.x dependencies for installing 2.0? If not, shouldn't this be on a 1.x tab?